Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG CADDY_VER=2.6.0
```

Comment:
Caddy 2.6.1 is being released currently, just FYI. The CI failed though due to timeout. We're trying to find a workaround...

(Would rather use 2.6.1 instead)

Reply:
Changed to 2.6.1